ext/mbstring/tests/koi8u_encoding.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Exhaustive test of verification and conversion of KOI8-U text
+--SKIPIF--
+<?php
+extension_loaded('mbstring') or die('skip mbstring not available');
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+?>
+--FILE--
+<?php
+srand(789); // Make results consistent
+include('encoding_tests.inc');
+mb_substitute_character(0x25); // '%'
+
+readConversionTable(realpath(__DIR__ . '/data/KOI8-U.txt'), $toUnicode, $fromUnicode);
```

Comment:
Just wondering, why the realpath() usage here?

Reply:
Maybe just superstition?

No, actually, I've remembered: Originally I had put these tables in a different folder, and `/../` was required, so I used `realpath` to squeeze out the `/../`.